mailpoet/lib/Subscribers/InactiveSubscribersController.php

```diff
@@ -160,40 +146,24 @@ private function activateSubscribers(Carbon $thresholdDate, int $batchSize): int
     $subscribersTable = $this->entityManager->getClassMetadata(SubscriberEntity::class)->getTableName();
     $connection = $this->entityManager->getConnection();
 
-    $mp2MigrationDate = $this->getMP2MigrationDate();
-    if ($mp2MigrationDate && $mp2MigrationDate > $thresholdDate) {
-      // If MP2 migration occurred during detection interval re-activate all subscribers created before migration
-      $idsToActivate = $connection->executeQuery("
-        SELECT id
-        FROM {$subscribersTable}
-        WHERE created_at < :migrationDate
-          AND status = :statusInactive
-        LIMIT :batchSize
-        ", [
-          'migrationDate' => $mp2MigrationDate,
-          'statusInactive' => SubscriberEntity::STATUS_INACTIVE,
-          'batchSize' => $batchSize,
-      ], ['batchSize' => \PDO::PARAM_INT])->fetchAllAssociative();
-    } else {
-      $idsToActivate = $connection->executeQuery("
-        SELECT s.id
-        FROM {$subscribersTable} s
-        LEFT OUTER JOIN {$subscribersTable} s2 ON s.id = s2.id AND GREATEST(
-          COALESCE(s2.last_engagement_at, '0'),
-          COALESCE(s2.last_subscribed_at, '0'),
-          COALESCE(s2.created_at, '0')
-        ) > :thresholdDate
-        WHERE s.last_subscribed_at < :thresholdDate
-          AND s.status = :statusInactive
-          AND s2.id IS NOT NULL
-        GROUP BY s.id
-        LIMIT :batchSize
-      ", [
-        'thresholdDate' => $thresholdDate,
-        'statusInactive' => SubscriberEntity::STATUS_INACTIVE,
-        'batchSize' => $batchSize,
-      ], ['batchSize' => \PDO::PARAM_INT])->fetchAllAssociative();
-    }
```

Comment:
We are removing the code for migrating subscribers, but the migrated subscribers will still remain in the database. So I guess we couldn't remove this code just because it is no longer possible to migrate subscribers. Also, there still be a flow for MP2 users in which they will be able to migrate (Download the last MP3 version that supports migration, migrate and then update).

But on the other hand, we have recently [strengthened the criteria for marking users inactive](https://mailpoet.atlassian.net/browse/MAILPOET-4081), and IMO, the new criteria "receiving three emails during the detection period" should cover also the case of migrated subscribers. So, in the end, I'm ok with removing the code. 

I'm posting this comment to explain the situation and so that someone can verify my reasoning. @rodrigoprimo Do you think the above makes sense?